emails/tests/views_tests.py

```diff
@@ -543,7 +550,10 @@ def test_reply(
         user = baker.make(User, email="source@sender.com")
         user.profile.server_storage = True
         user.profile.date_subscribed = datetime.now(tz=timezone.utc)
+        user.profile.last_engagement = datetime.now(tz=timezone.utc)
         user.profile.save()
+        pre_reply_last_engagement = user.profile.last_engagement
+        assert pre_reply_last_engagement is not None
```

Comment:
Nit: this assert is not needed

Reply:
Without it, I get this mypy error:
```
emails/tests/views_tests.py:378: error: Unsupported operand types for > ("datetime" and "None")  [operator]
emails/tests/views_tests.py:378: note: Right operand is of type "datetime | None"
```

Reply:
Yes, the one on line 378 is needed, but the one on line 556 is not.

Reply:
Whoops, wrong line. Okay checking thru ...